### PR TITLE
Add GetGroupMembers

### PR DESCRIPTION
**Pull request by jawwadmukhtargmail**

### CODE REVIEW

File: GroupService/GroupService.AzureFunction/GetGroupMembers.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        {
+            _mediator = mediator;
+            _logger = logger;
+        }
+
+        [FunctionName("GetGroupMembers")]
+        [ProducesResponseType((int)HttpStatusCode.OK, Type = typeof(GetGroupMembersResponse))]
+        public async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Function, "get", Route = null)]
+            [RequestBodyType(typeof(GetGroupMembersRequest), "get user groups")] GetGroupMembersRequest req,
```
Comment (mike-f50) [inline GroupService/GroupService.AzureFunction/GetGroupMembers.cs:36]: get group members?

File: GroupService/GroupService.Repo/Repository.cs [hunk 2]
```diff
@@ -157,5 +157,12 @@ public bool RoleAssigned(int userId, int groupId, GroupRoles groupRole, Cancella
                 return false;
             }
         }
+
+        public List<int> GetGroupMembers(GetGroupMembersRequest request, CancellationToken cancellationToken)
+        {
+            return _context.UserRole
+                .Where(w => w.GroupId == request.GroupID)
+                .Select(s => s.UserId).ToList();
```
Comment (mike-f50) [inline GroupService/GroupService.Repo/Repository.cs:165]: Does this need a .Distinct() ?

File: GroupService/GroupService.AzureFunction/GetGroupMembers.cs (hunk 1 above)
Comment (jawwadmukhtargmail) [reply GroupService/GroupService.AzureFunction/GetGroupMembers.cs]: changed

File: GroupService/GroupService.Repo/Repository.cs (hunk 2 above)
Comment (jawwadmukhtargmail) [reply GroupService/GroupService.Repo/Repository.cs]: done

**Pull request merged**